app/components/views/LNPage/NetworkTab/RoutesInfo.jsx

```diff
@@ -0,0 +1,57 @@
+import { FormattedMessage as T } from "react-intl";
+import { Subtitle, Balance } from "shared";
+import { CopyableText } from "pi-ui";
+import styles from "./NetworkTab.module.css";
+
+const Route = ({ route }) => (
+  <div className={styles.route}>
+    <span><T id="ln.routesInfo.routeAmount" m="Total amount" /></span>
+    <div className={styles.infoContent}><Balance amount={route.totalAmt} /></div>
+    <span><T id="ln.routesInfo.routeFees" m="Total fees" /></span>
+    <div className={styles.infoContent}><Balance amount={route.totalFees} /></div>
+    <div className={styles.hopsGrid}>
+      <span><T id="ln.routesInfo.hop" m="Hop" /></span>
+      <span><T id="ln.routesInfo.hopFee" m="Fee" /></span>
+      <span><T id="ln.routesInfo.hopPubkey" m="Public Key" /></span>
+      {route.hopsList.map((hop, i) => (
+        <>
```

Comment:
This needs a `key` prop, otherwise it triggers an error.

Reply:
I don't know how key works, but here I need to use the `<>` empty tag to ensure that elements don't render inside another `<div>`.
There is another away?

Reply:
https://reactjs.org/docs/fragments.html#keyed-fragments

Reply:
I fix it in another way, declaring a new component and use it with `key` props:
```
<Hop key={...} />
```